web_src/js/features/comp/TextExpander.ts

```diff
@@ -1,5 +1,38 @@
-import {matchEmoji, matchMention} from '../../utils/match.ts';
+import {matchEmoji, matchMention, matchIssue} from '../../utils/match.ts';
 import {emojiString} from '../emoji.ts';
+import {svg} from '../../svg.ts';
+
+type Issue = {state: 'open' | 'closed'; pull_request: {draft: boolean; merged: boolean} | null};
+function getIssueIcon(issue: Issue) {
```

Comment:
Duplicate code as `ContextPopup.vue`?

Reply:
Yep, extracted it from there. Will move it to a shared util.